### ISSUE TITLE
ARCHIVE_MODE setting not being honored

### ISSUE THREAD

**Issue by zyphermonkey** [splunk/splunk-connect-for-syslog#1206]
I have enabled archive mode in the global env file, but it's not honoring my ARCHIVE_MODE setting and using the folder/file structure laid out in the docs.

**env_file**
```
SC4S_ARCHIVE_GLOBAL=yes
SC4S_GLOBAL_ARCHIVE_MODE=compliance
```

The docs say my structure should be
`<archive mount>/${YEAR}/${MONTH}/${DAY}/${fields.sc4s_vendor_product}_${YEAR}${MONTH}${DAY}${HOUR}${MIN}.log`
But mine is following the diode mode and ignoring my setting in env_file. 
`<archive mount>/${.splunk.sourcetype}/${HOST}/$YEAR-$MONTH-$DAY-archive.log`

Furthermore switching `SC4S_GLOBAL_ARCHIVE_MODE` to `diode` doesn't switch the structure either.

**Comment by zyphermonkey** [comment]
@rfaircloth-splunk any update on this?

**Comment by rjha-splunk** [comment]
fixed in 2.34.2 @zyphermonkey
